helm-docs.sh

```diff
@@ -3,44 +3,22 @@
 set -euo pipefail
 
 SCRIPTPATH=$(dirname "$0")
-HELM_DOCS_VERSION="1.11.0"
+README_GENERATOR_VERSION="2.5.1"
+README_GENERATOR_EXE="readme-generator"
 
-case "$(uname -s)" in
-  Linux*)
-    machine=Linux
-    shasum=sha256sum
-    exe=helm-docs
-    ;;
-  Darwin*)
-    machine=Darwin
-    shasum=shasum
-    exe=helm-docs
-    ;;
-  MINGW64*)
-    machine=Windows
-    shasum=sha256sum
-    exe=helm-docs.exe
-    ;;
-esac
-
-function install_helm_docs {
-  curl -LO "https://github.com/norwoodj/helm-docs/releases/download/v${HELM_DOCS_VERSION}/helm-docs_${HELM_DOCS_VERSION}_${machine}_x86_64.tar.gz"
-  curl -L --output /tmp/checksums_helm-docs.txt "https://github.com/norwoodj/helm-docs/releases/download/v${HELM_DOCS_VERSION}/checksums.txt"
-  grep "helm-docs_${HELM_DOCS_VERSION}_${machine}_x86_64.tar.gz" /tmp/checksums_helm-docs.txt | $shasum -c -
-  mkdir -p "$SCRIPTPATH/bin"
-  tar -xf "helm-docs_${HELM_DOCS_VERSION}_${machine}_x86_64.tar.gz" "${exe}"
-  mv "${exe}" "$SCRIPTPATH/bin/"
-  rm "helm-docs_${HELM_DOCS_VERSION}_${machine}_x86_64.tar.gz"
-}
-
-if [ ! -f "$SCRIPTPATH/bin/${exe}" ] ; then
-  install_helm_docs
-elif [[ ! "$("$SCRIPTPATH/bin/${exe}" --version)" =~ .*"$HELM_DOCS_VERSION".* ]] ; then
-  install_helm_docs
-else
-  echo "Using '$("$SCRIPTPATH/bin/${exe}" --version)'"
+if ! hash "${README_GENERATOR_EXE}" 2>/dev/null; then
+  echo "readme-generator not installed. Installing..."
```

Comment:
```suggestion
  echo >&2 "readme-generator not installed. Installing..."
```

Write logs to stderr. This way you can always write other output that could be piped to other scripts on stdout without having the logs interfere.

Reply:
Done.